libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ActionScope.cs

```diff
@@ -45,12 +60,30 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
             }
         }
 
+        /// <summary>
+        /// Called when the dialog is _continued_, where it is the active dialog and the
+        /// user replies with a new activity.
```

Comment:
Could be redacted with more context?, I don't understand the part after the ',' `where it is the active dialog and the user replies with a new activity`

Reply:
Taken from base Dialog.cs ln99, I agree that the meaning is not particularly clear. Maybe something along the lines of "Resumes activity"?